src/main/resources/templates/editMATLABTestTask.ftl

```diff
@@ -74,3 +74,11 @@ extraUtility=addExecutableLink list=uiConfigSupport.getExecutableLabels('matlab'
             description="Specify a path relative to the working directory." /]
         [/@ui.bambooSection]
 [/@ui.bambooSection]
+
+[@ui.bambooSection]
+    [@ww.checkbox labelKey='matlab.options' name='optionsChecked'  toggle='true'/]
+        [@ui.bambooSection dependsOn='optionsChecked' showOn=true]
+            [@ww.textfield labelKey='matlab.options.field' cssClass="long-field" name="matlabOptions"
+                description="Specify MATLAB startup options."/]
```

Comment:
Where will this text appear? In the UI of the "Run MATLAB Tests" task (https://github.com/mathworks/matlab-bamboo-plugin/blob/main/CONFIGDOC.md#run-matlab-tests)? Is so, why is it terminated with a period? I don't see a period being used in the existing UI.

General comment: If a piece of text affects the UI in Jenkins, Bamboo, and TeamCity, would it be possible to include a screenshot of the proposed UI as well? This lets me review the text more accurately given the UI context. For example, I can see the placement of the new text and its associated heading, and whether it's consistent with other UI elements (e.g., in term of language constructs and capitalization).

Reply:
I believe that all of the descriptions are full sentences, for example the `htmlModelCoverage` above has the description `Specify a task relative to the working directory.`

And good suggestion for the UI elements. I'll try to get some screenshots added.

Reply:
# UI Screenshots
<img width="281" alt="image" src="https://github.com/mathworks/matlab-bamboo-plugin/assets/122639681/d10b648c-3bfa-42d6-9df0-e79362cb6bbb">
<img width="287" alt="image" src="https://github.com/mathworks/matlab-bamboo-plugin/assets/122639681/3af26dff-63e6-40b3-b61c-0ae8d5705051">
<img width="296" alt="image" src="https://github.com/mathworks/matlab-bamboo-plugin/assets/122639681/1eae2f4f-4d07-4571-baa4-392a0a08586b">

Reply:
Thank you, Sam. This is great help. Here are a few comments:

- I like the idea of placing the new feature at the end of the list. 
- Take a look at the text for **Source folder** in the **Run MATLAB Tests** task: https://github.com/mathworks/matlab-bamboo-plugin/blob/main/CONFIGDOC.md#run-matlab-tests. Here we use the verb phrase "Specify source folder". To remain consistent with the existing design, we need to have "Specify startup options" instead of "Startup options" next to the new check boxes.
- In the **Run MATLAB Tests** task's UI, **Startup options** appears to be part of the **Generate coverage artifacts** section. Can you add some white space to separate it from the coverage list?

Reply:
I think the best way to add white space would be to add a section title, however I can't think of anything that isn't redundant. Maybe something like "Customize MATLAB start up"?

Reply:
I don't think it deserves its own title. How about placing it between "Specify source folder" and "Filter tests"?

Reply:
Sounds good to me